tests/test_wheel.py

```diff
@@ -32,8 +32,7 @@
     ]
 )
 def example_wheel(request):
-    file_name = os.path.join(helpers.TESTS_DIR, request.param)
-    return wheel.Wheel(file_name)
+    return wheel.Wheel(os.path.join(helpers.TESTS_DIR, request.param))
```

Comment:
Nitpick: This change appears to me to be a no-op stylistic change. I guess it's fine, but also feels out of scope for this PR.